src/modules/Post/index.jsx

```diff
@@ -25,17 +26,84 @@ export default class Post extends DefaultTemplate {
     i18n: PropTypes.object.isRequired,
   }
 
-  static getGithubUrl(repo, filename, action) {
-    return `${repo}/${action}/master/content/${filename}`
+  // 2 cts autoprefixer
+  static renderCSSBackground(metadata) {
+    let image = metadata.image
+    if (
+      metadata.image === true ||
+      (!metadata.image && metadata.credit)
+    ) {
+      image = "index.jpg"
+    }
+
+    // default to just the image
+    const backgrounds = {
+      // "" => no need for prefix, see trick below
+      "": [
+        [
+          `url("${image}")`,
+          "no-repeat 50% 50% / cover",
+        ],
+      ],
+    }
+
+    // if there is a modification that need some prefix, we use a trick
+    if (metadata.linearGradient || metadata.radialGradient) {
+      const prefixes = ["", "-webkit-"]
+      prefixes.forEach((prefix) => {
+        const background = []
+        if (metadata.linearGradient) {
+          background.push([
+            `${prefix}linear-gradient(${metadata.linearGradient})`,
+            "repeat 0% 0%",
+          ])
+        }
+        if (metadata.radialGradient) {
+          background.push([
+            `${prefix}radial-gradient(${metadata.linearGradient})`,
+            "repeat 0% 0%",
+          ])
+        }
+        // only need the image if there will be both prefixed and unprefixed
+        // versions
+        if (image) {
+          background.push([
+            `url("${image}")`,
+            "no-repeat 50% 50% / cover",
+          ])
+        }
+        backgrounds[prefix] = background
+      })
+    }
+
+    return {
+      backgroundColor: metadata.color,
+      // we can not handle a css fallback with the same property name
+      // since a js object doesn't handle that like css rules set
+      // so here is the 2cts trick
+      // background-image + background-size AND background. Yep.
+      ...(
+        backgrounds["-webkit-"]
+        ? {
+          backgroundImage: backgrounds["-webkit-"].map(bg => bg[0]).join(", "),
+          backgroundSize: backgrounds["-webkit-"].map(bg => bg[1]).join(", "),
+        }
+        : {}
+      ),
```

Comment:
This works "server side" and I don't see we are likely to get this website isomorphic yet (you know this will demand "some" time) :/
Do you really think we will benefit handling that this way right now ?